src/components/ReportCategory/ReportCategory.test.tsx

```diff
@@ -107,4 +75,212 @@ describe('ReportCategory', () => {
       expect(screen.getByAltText("location-illustration-1")).toBeInTheDocument();
   });
 
+  it('should render with titles', () => {
+    const fakeUserReport = {
+      zip_code: 11111,
+      location: "Anywhere, CO"
+    }
+
+    const fakeReportData = Object.keys(fakeData);
+    const fakeSubtitle_2 = '_location';
+    const fakeKey = 0;
+
+    render (
+      <ReportCategory
+      position={fakeKey + 1}
+      key={fakeKey}
+      plan={fakeData.downpayment.ten_year_plan}
+      categoryName={fakeReportData[0]}
+      categoryMainNumber={fakeUserReport.zip_code || fakeData.principal.based_on_rent
+      || fakeData.monthly.monthly_principal
+      || fakeData.downpayment.down_payment_percentage_selected}
+      categoryMainTitle={'location'}
+      categorySubtitle={fakeSubtitle_2}
+      categorySecondNumber={fakeUserReport.location || fakeData.principal.goal_principal
+      || fakeData.monthly.estimated_true_monthly
+      || fakeData.downpayment.down_payment_saved}
+      categoryID={fakeKey + 1}
+      /> );
+
```

Comment:
Good job figuring this out, was thinking it was going to be a huge pain, glad you got it working!